tests/testthat/test-tidyeval.R

```diff
@@ -19,7 +19,6 @@ test_that("existing non-variables get inlined", {
 
   n <- 10
   expect_equal(capture_dot(lf, x + n), expr(x + 10))
-  expect_equal(capture_dot(lf, x + m), expr(x + m))
```

Comment:
I don't think this test made much sense.

Reply:
Agreed. I'm not sure why I made it work this way. The only reason I can think of is to make it possible to refer to database specific constants, but that seems like a pretty weak argument.